com.microsoft.java.test.runner/src/main/java/com/microsoft/java/test/runner/TestMessageItem.java

```diff
@@ -0,0 +1,38 @@
+package com.microsoft.java.test.runner;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Arrays;
+import java.util.List;
+
+public class TestMessageItem {
+    String name;
+
+    List<Pair> attributes;

```

Comment:
Why not use hashmap?

Reply:
if using a hashmap, when creating an instance of TestMessageItem, it would introduce a lot of code blocks to create a hashmap and then set the key value pair. So I just use a list instead.